### DISCUSSION TITLE
Class "Laravel\Nova\Menu\MenuSeparator" not found

### DISCUSSION THREAD

**Discussion by alexandru-eftimie** [laravel/nova-issues#5114] [Q&A]
- Laravel Version: v9.41.0
- Nova Version: 4.19.2
- PHP Version: 8.1.12
- Database Driver & Version:
- Operating System and Version: macos 12.6.1
- Browser type and version: chrome 107.0.5304.110
- Reproduction Repository: https://github.com/###/###

### Description: 
Worked fine on 4.18.2, it displayed a small spacer in the user menu.

**Comment by crynobone** [comment]
`MenuSeparator` has been removed since 4.19.0

You can however replicate the feature directly on your app.

```php
<?php

namespace App\Nova\Menu;

use Laravel\Nova\Makeable;

class MenuSeparator implements \JsonSerializable
{
    /**
     * Prepare the menu for JSON serialization.
     *
     * @return array<string, mixed>
     */
    public function jsonSerialize(): array
    {
        return [
            'component' => 'custom-menu-separator',
        ];
    }
}
```

```js
Nova.booting(app => {
  app.component('CustomMenuSeparator', require('./components/CustomMenuSeparator').default)
})
```

```js
<template>
  <div class="sidebar-separator">
    <hr class="border-none bg-50" style="height: 1px" />
  </div>
</template>

<script>
export default {
  props: ['item'],
}
</script>
```

**Reply by alibori** [reply]
Hi @crynobone !

Please, can you detail a little bit more the steps to achieve this?

Thank you so much!

**Comment by fruitl00p** [comment]
Hmmm... would've been nice to have this mentioned in the changelog somewhere ;)
